src/App.tsx

```diff
@@ -151,6 +155,11 @@ const hasuraOptions: HasuraDataProviderOptions = {
   },
 };
 
+const hasuraDataProvider = dataProvider(client, hasuraOptions)
+hasuraDataProvider.getApiUrl = () => {
```

Comment:
why is this needed? 
```
hasuraDataProvider.getApiUrl = () => {
  return API_URL;
}
```

Reply:
otherwise the method is not implemented.